capa/render/proto/capa.proto

```diff
@@ -179,22 +189,22 @@ message MaecMetadata {
 message Match {
   bool success = 1;
   oneof node {
-    StatementNode v0 = 2;
-    FeatureNode v1 = 3;
+    StatementNode statement = 2;
+    FeatureNode feature = 3;
   };
   repeated Match children = 5;
   repeated Address locations = 6;
-  map <string, Array_Address> captures = 7;
+  map <string, Addresses> captures = 7;
 }
 
 message MatchFeature {
   string type = 1;
   string match = 2;
-  string description = 3;
+  optional string description = 3;
 }
 
 message Metadata {
-  string timestamp = 1;
+  string timestamp = 1;  // google.protobuf.timestamp_pb2.Timestamp also would work, but seems more of a headache
```

Comment:
agree. but lets specify the format of the string as comments here, maybe ISO8601, like "2023-03-21T01:02:03.004Z"